### PR TITLE
docs(quickstart): update with changes from last release

### LINKED ISSUE #1484: Quickstart installation fails
**Describe the bug**

I just tried following ./opstrace create aws nagyv-opstrace-test \
  -c opstrace-config.yaml

**To Reproduce**

Follow the quickstart

```bash
./opstrace create aws nagyv-opstrace-test \
  -c opstrace-config.yaml
2021-11-25T14:32:54.792Z info: logging to file: opstrace_cli_create_20211125-143254Z.log
2021-11-25T14:32:54.793Z info: Discovered AWS credentials. Access key: AKIA...77AP
2021-11-25T14:32:54.799Z error: invalid instance config document: custom_auth0_domain is a required field
```

**Expected behavior**

An Opstrace cluster is created in the AWS account.

### PR DESCRIPTION
**Pull request by triclambert**
Resolves #1484.

The use of `example.com` here breaks all of the links.  Is that ok?  Should we try to use the same substitution mechanism we have for the instance name?  Thoughts?

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -81,7 +89,7 @@ export OPSTRACE_NAME=<choose_a_name>
 
 <!--/export-to-input-->
 
-The name will globally identify you in our domain as `$OPSTRACE_NAME.opstrace.io`, which we provide for you by default as a convenience.
+The name will globally identify you in our domain as `$OPSTRACE_NAME.example.com`, which we provide for you by default as a convenience.
```
Comment (sreis) [inline docs/quickstart.md:92]:
```suggestion
The name will globally identify your Opstrace instance as `$OPSTRACE_NAME.example.com`.
```

File: docs/quickstart.md [hunk 2]
```diff
@@ -67,7 +67,15 @@ curl -L https://go.opstrace.com/cli-latest-release-linux | tar xjf -
...
-So let's get started.
+
+### Authentication
+
+Opstrace is secure by default—it requires authentication (provided through Auth0) to work.
+It's easy to get started—follow [this guide to set up Auth0](https://github.com/opstrace/opstrace/wiki/User-guide:-custom-Auth0-integration).
+
+### DNS
+
+Next, create DNS records to point to your cluster by [following these instructions](https://github.com/opstrace/opstrace/wiki/User-guide:-Upgrade-Opstrace-with-a-custom-DNS-domain-and-custom-Auth0-integration).
```
Comment (sreis) [inline docs/quickstart.md:78]:
```suggestion
Next, set up a DNS domain for your Opstrace instance by [following these instructions](https://github.com/opstrace/opstrace/wiki/User-guide:-Upgrade-Opstrace-with-a-custom-DNS-domain-and-custom-Auth0-integration).
```

File: docs/quickstart.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ Open a terminal and verify you have the following:
   * While GCP is also fully supported, we will focus on AWS in this quick start.
 * For sending dummy data to Opstrace, you'll need [Docker](https://docs.docker.com/install) and [Docker Compose](https://docs.docker.com/compose/install)
 
-Note: the code blocks are all copy-and-pastable.
+You will also need an Auth0 account to wire up Google or GitHub log in (sign up for free here).
```
Comment (sreis) [inline docs/quickstart.md:27]: Needs a link.

**Review by sreis — COMMENTED** [review]
I'm ok with the `example.com` for now. We can always change it later based on user feedback. 

Left a couple of comments.

**Review by jgehrcke — CHANGES_REQUESTED** [review]
When specifying `custom_dns_name: instance.example.com` then `Log in here: https://$OPSTRACE_NAME.example.com` is wrong.

With the introduction of `custom_dns_name` there is no relationship anymore between what is called `OPSTRACE_NAME` here and the DNS name pointing to the Opstrace instance.

See  maybe https://opstrace.com/docs/references/configuration#custom_dns_name for a simpler way to describe this.

Quote:

> This DNS name will point to the specific Opstrace instance you are planning to create. For example, the UI will then be served under `https://<custom_dns_name>/`.

File: docs/quickstart.md (hunk 3 above)
Comment (triclambert) [reply docs/quickstart.md:27]: Removing this line in favor of the authentication section below.

**Comment by triclambert** [comment]
Testing the substitution here:  https://preview.opstrace.com/1488/docs/quickstart.html#step-1-install-opstrace

**Review by sreis — APPROVED** [review]

**Comment by triclambert** [comment]
@jgehrcke will leave this here for you to re-review and then either update or merge.  Thanks.

**Pull request merged**